config/locales/en.yml

```diff
@@ -292,6 +297,10 @@ en:
           any_humanities_subject: Any humanities subject
           any_science_subject: Any science subject
           other_subject: Choose a subject
+      add_a_level_to_a_list:
+        add_another_a_level_options:
+          "yes": "Yes"
+          "no": "No"
```

Comment:
```suggestion
          yes: Yes
          no: No
```

Reply:
actually, if I remove the quotes it won't work as YAML will parse as true or false.